### PR TITLE
Michael Amparo

### PR DESCRIPTION
**Pull request by mike923**

### CODE REVIEW

**Comment by MikePursuit** [comment]
couldnt find a better way of doing #3 without using the function on my last edit

File: loop_lab.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+let max = 10;
+
+while (max > 0) {
+  if (max % 2 === 0) {
+    console.log(max + ' is even');
+  } else if (max % 2 === 1) {
+    console.log(max + ' is odd');
+  } else {
+    console.log('error ' + max + ' is not an integer.');
+  } max--;
```
Comment (jungraejang) [inline loop_lab.js]: I think the question was asking for ascending numbers from max, not descending from max. Minor thing really. Good implementation of while and for loops.

File: loop_lab.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+}
+
+for (let i = 0; i <= x * 9; i += 9) {
+  console.log(i);
+}
+
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+
+for (let i = 1; i <= 100; i++) {
+  i % 5 === 0 && i % 2 === 1 ? console.log(i) : null;
```
Comment (jungraejang) [inline loop_lab.js]: Very interesting implementation using modulo 2 === 1.

File: loop_lab.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+//     Hint 2
+//   </summary>
+//   You will need 2 loops for this.
+// </details>
+
+let n = 5;
+let str = '';
+for (let i = 2 * n; i > 0; i--) {
+  i > n ? str += '*' : console.log(str);
+}
```
Comment (jungraejang) [inline loop_lab.js]: Overall good implementation of for and while loops. Keep up the good work!

**Pull request closed without merging**